packages/presets/src/fragments/copilot-panel/edgeless/logic.ts

```diff
@@ -39,36 +41,37 @@ export class AIEdgelessLogic {
   public get autoGen() {
     return this.unsub !== undefined;
   }
+
   private unsub?: () => void;
   public toggleAutoGen = () => {
     if (this.unsub) {
       this.unsub();
       this.unsub = undefined;
       return;
     }
-    const edgeless = getEdgelessPageBlockFromEditor(this.editor.host);
+    const edgeless = getEdgelessPageBlockFromEditor(this.host);
```

Comment:
I think this kind of util should be avoided, or you can just expose a getter on `edgeless-page-service` - and you can access the service here.

What are the APIs that are being used here? I think they should also be defined as standard methods on edgeless page service, cc @doouding

Reply:
How do I get `edgeless-page-service`?